setup.py

```diff
@@ -43,5 +43,6 @@
         "numpy",
         "tensorflow>=2.4",
         "tensorflow-probability>=0.12,<0.14.0",
+        "greenlet>1.1.0",
```

Comment:
slightly surprising, this is usually using `>=` because you are relying on a certain version of the package, and up from there. why is this strictly greater then?